doc/user-guide.md

```diff
@@ -107,7 +107,22 @@ After modifying the `*_types.go` file always run the following command to update
 $ operator-sdk generate k8s
 ```
 
-### OpenAPI validation
+### Updating CRD manifests
+
+Now that `MemcachedSpec` and `MemcachedStatus` have fields and possibly annotations, the CRD corresponding to the API's group and kind must be updated. To do so, run the following command:
+
+```console
+$ operator-sdk generate crds
+```
+
+**Notes:**
+- - Your CRD *must* specify exactly one [storage version][crd-storage-version]. Use the `+kubebuilder:storageversion` [marker][crd-markers] to indicate the GVK that should be used to store data by the API server. This marker should be in a comment above your `Memcached` type.
```

Comment:
I noticed that without manually adding `+kubebuilder:storageversion` in advance, `operator-sdk add api` errors out when another version is added to an existing CRD with the same group & kind.

This problem may be out-of-scope for this PR, but something we may want to think about whether there's a more graceful way to handle from a user experience perspective.

Here's the error I get:
```
$ operator-sdk generate crds
INFO[0000] Running CRD generator.
example.com/go-operator/pkg/apis/example/v1:-: CRD for GoOperator.example.com has no storage version
Error: error generating CRDs from APIs in pkg/apis: error generating CRD manifests: error running CRD generator: not all generators ran successfully
Usage:
  operator-sdk generate crds [flags]

Flags:
  -h, --help   help for crds

Global Flags:
      --verbose   Enable verbose logging
```

It's a good hint, but could be improved by telling the user how to fix the problem. Can we put this note into the error message, nearly verbatim and skip outputting the usage?

Reply:
Also, I just noticed that it's possible to define `+kubebuilder:storageversion` in both versions of a CRD's types.go files and controller-gen happily creates a CRD where `storage: true` exists in both versions. That should also be an error right?

Reply:
:+1: on not printing usage. By note do you mean:
```
example.com/go-operator/pkg/apis/example/v1:-: CRD for GoOperator.example.com has no storage version
```

Two CRD versions with `storage: true` is an error, but controller-tools does not consider this an error during generation. I will submit an issue if one does not exist already upstream.

Reply:
Oh I meant based on the note that you have in the user guide:

Something like:
> Your CRD must specify exactly one storage version. Use the `+kubebuilder:storageversion` marker to indicate the GVK that should be used to store data by the API server. This marker should be in a comment above your CRDs root storage type in `_types.go`.
>
> See https://book.kubebuilder.io/multiversion-tutorial/api-changes.html#storage-versions

Reply:
Generator errors are not returned by [`Runtime.Run()`](https://github.com/kubernetes-sigs/controller-tools/blob/b62a514/pkg/genall/genall.go#L172) so we can't check if the error returned pertains to no `+kubebuilder:storageversion` annotation. How about linking https://book.kubebuilder.io/reference/generating-crd.html in an error message for `Runtime.Run()` errors? This should cover this and other errors, but isn't as specific.